docs/tokens.md

```diff
@@ -0,0 +1,126 @@
+# Tokens
+
+The `Tokens` class (`from mapbox import Tokens`) provides
+access to the Mapbox Tokens API, allowing you to programmaticaly create
+Mapbox access tokens to access Mapbox resources on behalf of a user.
+
+```python
+
+>>> from mapbox import Tokens
+>>> service = Tokens()
+
+```
+
+See https://www.mapbox.com/api-documentation/#tokens for general documentation of the API.
+
+This API requires an **initial token** with the `tokens:write` scope.
+Your Mapbox access token should be set in your environment;
+see the [access tokens](access_tokens.md) documentation for more information.
+
+The Mapbox username associated with each account is determined by the access_token by default. All of the methods also take an optional `username` keyword argument to override this default.
```

Comment:
Yeah, we should make that consistent. But looks like the main API docs use `username`throughout; maybe the solution is to make the change in uploads?